core/src/main/java/io/questdb/PropertyKey.java

```diff
@@ -473,6 +473,7 @@ public enum PropertyKey implements ConfigPropertyKey {
     CAIRO_REPEAT_MIGRATION_FROM_VERSION("cairo.repeat.migration.from.version"),
     CAIRO_O3_LAST_PARTITION_MAX_SPLITS("cairo.o3.last.partition.max.splits"),
     CAIRO_O3_PARTITION_SPLIT_MIN_SIZE("cairo.o3.partition.split.min.size"),
+    CAIRO_O3_PARTITION_OVERWRITE_CONTROL_ENABLE("cairo.o3.partition.overwrite.control.enable"),
```

Comment:
nit: in all other boolean props we use `.enabled`, not `enable`.

Reply:
changed to `enabled`